dotnet/src/Connectors/Connectors.Ollama/OllamaPromptExecutionSettings.cs

```diff
@@ -46,5 +47,6 @@ public static OllamaPromptExecutionSettings FromExecutionSettings(PromptExecutio
     /// <summary>
     /// Maximum number of tokens to generate.
     /// </summary>
+    [JsonPropertyName("max_tokens")]
     public int? MaxTokens { get; set; }
```

Comment:
adjust prompt execution settings to new pattern used actual in openai connector.